packages/dai-plugin-liquidations/src/LiquidationService.ts

```diff
@@ -15,11 +16,9 @@ export const RAY = new BigNumber('1e27');
 
 export const nullBytes = '0x';
 
-export const stringToBytes = str => {
-  assert(!!str, 'argument is falsy');
-  assert(typeof str === 'string', 'argument is not a string');
-  return '0x' + Buffer.from(str).toString('hex');
-};
+export function stringToBytes(str) {
+  return utils.formatBytes32String(str);
```

Comment:
Ethers is more strict about types, in some places where we used to pass a function signature as `0x4554482d41`, we now have to pass the full 32 bytes, eg. `0x4554482d41000000000000000000000000000000000000000000000000000000`. Fortunately ethers utils has a built in function for that.

this conversion pads the string to 32 bytes correctly